stablehlo/tests/infer_subtract.mlir

```diff
@@ -0,0 +1,258 @@
+// RUN: stablehlo-interpreter --interpret -split-input-file %s | FileCheck %s
```

Comment:
Can't we reuse the same constant as in add? Only the expected outputs would be different.

Reply:
Hmm, how?

Reply:
I think he meant to say to reuse the values from lhs and rhs of add op. Will fix.